uber/templates/base.html

```diff
@@ -295,14 +295,48 @@
                 });
             };
 
-            $('.datatable').dataTable({
-                aLengthMenu: [
-                    [25, 50, 100, 200, -1],
-                    [25, 50, 100, 200, 'All']
-                ],
-                stateSave: true,
-                drawCallback: createDateTextEntries
-            });
+                $('.datatable').each(function() {
```

Comment:
This allows us to add dropdown filters to dataTable tables if we so desire. I think it may be slightly buggy, though... I seem to recall that the filter doesn't reset if you navigate away from the page.